services/backend/src/router/profile/profile.js

```diff
@@ -13,11 +17,7 @@ profileRouter
     [UUIDValidator, langValidator],
     profile.getPublicProfileById
   )
-  .put(keycloak.protect(), profile.updateProfile);
-
-profileRouter
-  .route("/:id/status")
-  .get(keycloak.protect(), UUIDValidator, profile.getProfileStatusById);
+  .put(keycloak.protect(), updateProfileValidator, profile.updateProfile);
```

Comment:
we should also add UUID validation for the put request.

Reply:
same as below